bin/cdxgen.js

```diff
@@ -388,7 +388,7 @@ if (process.env.GLOBAL_AGENT_HTTP_PROXY || process.env.HTTP_PROXY) {
   globalAgent.bootstrap();
 }
 
-const filePath = args._[0] || process.cwd();
+const filePath = resolve(args._[0]) || process.cwd();
```

Comment:
Use of `resolve` would convert everything to absolute path. Hope this would be alright.